doc/source/dev/conduct/code_of_conduct.rst

```diff
@@ -0,0 +1,175 @@
+   harassment or other exclusionary behaviour, such as:
+
+  - Violent threats or language directed against another person.
+  - Sexist, racist, or otherwise discriminatory jokes and language.
+  - Posting sexually explicit or violent material.
+  - Posting (or threatening to post) other people's personally identifying information ("doxing").
+  - Sharing private content, such as emails sent privately or non-publicly,
+    or unlogged forums such as IRC channel history, without the sender's consent.
+  - Personal insults, especially those using racist or sexist terms.
+  - Unwelcome sexual attention.
+  - Excessive profanity. Please avoid swearwords; people differ greatly in their sensitivity to swearing.
+  - Repeated harassment of others. In general, if someone asks you to stop, then stop.
+  - Advocating for, or encouraging, any of the above behaviour.
+
+
+Diversity Statement
+-------------------
+
+The PyWavelets project welcomes and encourages participation by everyone. We
+are committed to being a community that everyone enjoys being part of. Although
+we may not always be able to accommodate each individual's preferences, we try
+our best to treat everyone kindly.
+
+No matter how you identify yourself or how others perceive you: we welcome you.
+Though no list can hope to be comprehensive, we explicitly honour diversity in:
+age, culture, ethnicity, genotype, gender identity or expression, language,
+national origin, neurotype, phenotype, political beliefs, profession, race,
+religion, sexual orientation, socioeconomic status, subculture and technical
+ability, to the extent that these do not conflict with this code of conduct.
+
+Though we welcome people fluent in all languages, PyWavelets development is
+conducted in English.
+
+Standards for behaviour in the PyWavelets community are detailed in the Code of
+Conduct above. Participants in our community should uphold these standards
+in all their interactions and help others to do so as well (see next section).
+
+
+Reporting Guidelines
+--------------------
+
+We know that it is painfully common for internet communication to start at or
+devolve into obvious and flagrant abuse.  We also recognize that sometimes
+people may have a bad day, or be unaware of some of the guidelines in this Code
+of Conduct. Please keep this in mind when deciding on how to respond to a
+breach of this Code.
+
+For clearly intentional breaches, report those to the Code of Conduct committee
+(see below). For possibly unintentional breaches, you may reply to the person
+and point out this code of conduct (either in public or in private, whatever is
+most appropriate). If you would prefer not to do that, please feel free to
+report to the Code of Conduct Committee directly, or ask the Committee for
+advice, in confidence.
+
+You can report issues to the PyWavelets Code of Conduct committee, at
+PyWavelets-conduct@googlegroups.com. Currently, the committee consists of:
+
+- Gregory Lee
+- Ralf Gommers
```

Comment:
we may want to invite a third person here, perhaps from scikit-image since they rely on pywavelets and likely someone on that team will be happy to help out?

Reply:
Should we also include the committee member's email addresses here? (or were those deliberately omitted to reduce spam?)

Reply:
It shouldn't be needed I think. And if someone really wants to find us, they can look at the commit log so it's not impossible. I'd leave it out here.